src/styles/components/_related-components.scss

```diff
@@ -38,14 +62,17 @@ $breakpoint-tablet: 900px;
   &__link {
     text-decoration: underline;
     transition: all ease-in-out 100ms;
+    outline: none;
```

Comment:
Were you instructed to remove the outline for links? Typically we don't want to remove the outline because it takes away the focus indicator, which isn't great for accessibility

Reply:
Got it- no we were not. That was something I've done before, but honestly these hovers aren't the same as the last time I did that, so it makes more sense to keep the outline on. I'll get that removed and let you know when I push it up! 

Also to answer your question above (I don't know- there was no option for me to reply 🤷‍♀️): yes. I have that lint rule disabled because we do/might have hidden content in those code blocks. Adding the `tabIndex` was a suggested fix from AxeDevTools (to fix the error that the scrolling container wasn't keyboard accessible), but then I got linting errors about not putting `tabIndex` on non-interactive items. I would think scrolling constitutes "interactive", so I added that disable for the one line. Is there another, better way to achieve the same thing?

Reply:
Awesome! And no I think that's the way to do it if these are going to be scrollable